Library/Tasks/AbstractTask.php

```diff
@@ -113,14 +114,16 @@ private function parseOpt(array $paramList)
         $this->lock();
         $this->initWorker();
 
-        $this->kafkaTopicConsumer = $this->kafkaConsumer->newTopic($this->topicName, $this->kafkaTopicConf);
-        $this->kafkaTopicConsumer->consumeStart($this->topicPartitionNo, RD_KAFKA_OFFSET_STORED);
         // xxx Factorize stdout/err support
-        $this->getDI()->get('logger')->info("Worker listening on \033[32m" .
+        $this->getDI()->get('logger')->info(
+            "Worker listening on \033[32m" .
             implode(',', $this->workflowConfig['brokerServerList']->toArray()) .
-            ":\033[32m" . $this->topicName . "\033[0m");
+            ":\033[32m" . $this->topicName . "\033[0m"
+        );
+        $this->kafkaConsumer->subscribe([$this->topicName]);
         while (true) {
-            $msg = $this->kafkaTopicConsumer->consume($this->topicPartitionNo, 100);
+            sleep(1);
+            $msg = $this->kafkaConsumer->consume(10000);
```

Comment:
10000 ?

Reply:
right this is a timeout, still have to deal w/